planet/scripts/cli.py

```diff
@@ -200,23 +200,49 @@ async def cancel(ctx, order_id):
 @click.argument('order_id', type=click.UUID)
 @click.option('-q', '--quiet', is_flag=True, default=False,
               help=('Disable ANSI control output.'))
-@click.option('-o', '--overwrite', is_flag=True, default=True,
+@click.option('-o', '--overwrite', is_flag=True, default=False,
               help=('Overwrite files if they already exist.'))
 @click.option('--dest', default='.',
               help=('Directory to download files to.'),
               type=click.Path(exists=True, resolve_path=True,
                               writable=True, file_okay=False))
+                              
 async def download(ctx, order_id, quiet, overwrite, dest):
     '''Download order by order ID.'''
-    async with orders_client(ctx) as cl:
-        await cl.poll(str(order_id), verbose=True)
-        filenames = await cl.download_order(
-                str(order_id),
-                directory=dest,
-                overwrite=overwrite,
-                progress_bar=not quiet)
-    click.echo(f'Downloaded {len(filenames)} files.')
 
+    if overwrite:
+        # Explicity ask the user if they want to overwrite their data
+        overwrite_decision = click.prompt(f'You have selected to overwrite your data. Proceed? [y/n]', type=str).lower()
```

Comment:
Adding the overwrite flag seems like assurance enough that the user wants to overwrite the files. Prompts like this are more work for anyone wanting to use scripting so I tend to avoid them unless absolutely necessary.

Reply:
I'm not against removing this prompt, I mostly added it as a backup plan, if the user didn't mean to select to overwrite their data. My only concern was that the user may have hit the up arrow a few times in their terminal and accidentally selected to overwrite all the data they may have spent an hour downloading. We could keep it and add a `-f` / `--force` flag, which would skip this prompt.

Thoughts?

Reply:
We already have a flag they have to add to overwrite. Adding a force flag is just requiring two flags for one action (and, honestly, is subject to the same user error). I have trust in the user that they want what they say they want. I suggest getting rid of the prompt all together.

Reply:
Cool - agreed!